[2023-10-20 23:20:56,671] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-17T00:00:00+00:00 [queued]>
[2023-10-20 23:20:56,683] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-17T00:00:00+00:00 [queued]>
[2023-10-20 23:20:56,684] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:20:56,686] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:20:56,687] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:20:56,709] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-10-17 00:00:00+00:00
[2023-10-20 23:20:56,715] {standard_task_runner.py:52} INFO - Started process 924 to run task
[2023-10-20 23:20:56,721] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-10-17T00:00:00+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpmlyqe0wn', '--error-file', '/tmp/tmp_mqk1_88']
[2023-10-20 23:20:56,724] {standard_task_runner.py:80} INFO - Job 235: Subtask segunda_tarea
[2023-10-20 23:20:56,804] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-17T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:20:56,884] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-10-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-17T00:00:00+00:00
[2023-10-20 23:20:56,886] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:20:56,886] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:20:56,895] {subprocess.py:85} INFO - Output:
[2023-10-20 23:20:56,897] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:20:56,897] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:20:56,925] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221017T000000, start_date=20231020T232056, end_date=20231020T232056
[2023-10-20 23:20:56,973] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:20:57,061] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
